newIDE/app/src/Utils/GDevelopServices/Project.spec.js

```diff
@@ -2,88 +2,120 @@
         )
       ).toBe('Pea-Happy');
       expect(
-        extractFilenameFromProjectResourceUrl(
+        extractDecodedFilenameFromProjectResourceUrl(
           'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea-Happy.png'
         )
       ).toBe('Pea-Happy.png');
       expect(
-        extractFilenameFromProjectResourceUrl(
+        extractDecodedFilenameFromProjectResourceUrl(
           'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea-Happy?some=parameter'
         )
       ).toBe('Pea-Happy');
       expect(
-        extractFilenameFromProjectResourceUrl(
+        extractDecodedFilenameFromProjectResourceUrl(
           'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea-Happy.png?some=parameter'
         )
       ).toBe('Pea-Happy.png');
     });
     it('extracts filename from a gdevelop.io bucket URL, and handles sub folders', () => {
       expect(
-        extractFilenameFromProjectResourceUrl(
+        extractDecodedFilenameFromProjectResourceUrl(
           'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/some/other-folder/1/2/_/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea-Happy'
         )
       ).toBe('Pea-Happy');
       expect(
-        extractFilenameFromProjectResourceUrl(
+        extractDecodedFilenameFromProjectResourceUrl(
           'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/some/other-folder/1/2/_/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea-Happy.png'
         )
       ).toBe('Pea-Happy.png');
       expect(
-        extractFilenameFromProjectResourceUrl(
+        extractDecodedFilenameFromProjectResourceUrl(
           'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/some/other-folder/1/2/_/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea-Happy?some=parameter'
         )
       ).toBe('Pea-Happy');
       expect(
-        extractFilenameFromProjectResourceUrl(
+        extractDecodedFilenameFromProjectResourceUrl(
           'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/some/other-folder/1/2/_/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea-Happy.png?some=parameter'
         )
       ).toBe('Pea-Happy.png');
     });
+    it('extracts filename from a gdevelop.io bucket URL, if encoded', () => {
+      expect(
+        extractDecodedFilenameFromProjectResourceUrl(
+          'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/some/other%20folder/1/2/_/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea Happy'
```

Comment:
Should it be `'https://project-resources.gdevelop.io/a9fe5bce-de39-4147-a669-93fc5cd69632/resources/some/other%20folder/1/2/_/6ef87bc678921eb4bfa2d04e5dc6a16b75f7b239f3163d0c5efe64d4cc501711-Pea%20Happy'`?

Reply:
yes indeed